pkg/api/client/amqp/qeclients/argparser/java/parser_test.go

```diff
@@ -0,0 +1,15 @@
+package java
+
+import (
+	"reflect"
+	"testing"
+)
+
+// TestUrl validates the Url function for the cli-java parser
+func TestUrl(t *testing.T) {
+	args := Url("amqp://user:pass@127.0.0.1:5672/my/address/name")
```

Comment:
what is expected to happen if you call it with corrupted url? maybe the test can  also call the function with malformed, and other kinds of incorrect urls.

Reply:
https://github.com/rh-messaging/shipshape/pull/

Reply:
great!